test/powershell/Language/Scripting/OutErrorVariable.Tests.ps1

```diff
@@ -0,0 +1,390 @@
+﻿function Validate-Result {
+    param($out, $outStr, $count, $v)
+
+    It "$outStr.count" { $out.count | Should Be $count }
+    for($i=0; $i -lt $count; $i++)
+    {
+        It "$outStr[$i]" { $out[$i] | Should Be $v[$i] } 
+    }                
+}
+
+Describe "Tests OutVariable only" -Tags "CI" {
+    BeforeAll {
+
+        function get-foo1
+        {
+            [CmdletBinding()]
+            param()
+
+            "foo"
+        }
+
+        function get-foo2
+        {
+            [CmdletBinding()]
+            param()
+
+            $pscmdlet.writeobject("foo")
+        }
+
+        function get-bar 
+        {
+            [CmdletBinding()]
+            param()
+
+            "bar"
+            get-foo1 -outVariable global:a
+        }
+    }
+
+    Context 'Updating OutVariable Case 1: pipe string' {        
+
```

Comment:
fixed

---
In reply to: [77278055](https://github.com/Powershell/powershell/pull/2160#discussion_r77278055) [](ancestors = 77278055)